src/inc/msquicp.h

```diff
@@ -78,6 +98,7 @@ typedef struct QUIC_PRIVATE_TRANSPORT_PARAMETER {
 #define QUIC_PARAM_CONN_FORCE_KEY_UPDATE                0x80000001  // No payload
 #define QUIC_PARAM_CONN_FORCE_CID_UPDATE                0x80000002  // No payload
 #define QUIC_PARAM_CONN_TEST_TRANSPORT_PARAMETER        0x80000003  // QUIC_PRIVATE_TRANSPORT_PARAMETER
+#define QUIC_PARAM_CONN_SSLKEYLOG_BUFFER                0x80000004  // Set-only, sizeof(QUIC_SSLKEYLOG)
```

Comment:
I don't think we call out "set-only" in any of the other comments. Just put `QUIC_SSLKEYLOG` here,